shared/oae/bundles/email/default.properties

```diff
@@ -8,4 +11,8 @@ ACTIVITY_EMAIL_SUMMARY_IMMEDIATE_MULTIPLE_ACTORS = New activity is waiting for y
 ACTIVITY_EMAIL_SUMMARY_IMMEDIATE_SINGLE_ACTOR_MULTIPLE_ACTIVITIES = ${actorLink} has been very active recently
 ACTIVITY_EMAIL_SUMMARY_IMMEDIATE_SINGLE_ACTOR_SINGLE_ACTIVITY = ${actorLink} has been active recently
 ACTIVITY_EMAIL_SUMMARY_WEEKLY = Here's what happened in the last week
-
```

Comment:
Email doesn't have to be enabled. The email debug message that gets put in the log has the link that can be copy/pasted for verifying your email.